test_js/float.test.ts

```
@@ -0,0 +1,70 @@
+import { Float } from '../dist/cjs';
+import { describe, it, expect, assert } from 'vitest';
+
+describe('Test Float Bindings', async function () {
+	it('should try from bigint', async function () {
+		const result = Float.tryFromBigint(5n);
+		assert.ok(result.error === undefined, `Expected no error, but got: ${result.error?.readableMsg}`);
+
+		const expected = Float.fromHex('0x0000000000000000000000000000000000000000000000000000000000000005')?.value!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)